Dockerfile

```diff
@@ -31,9 +31,10 @@ RUN npm install
 RUN echo "Starting the script section" && \
     pub get && \
     dartanalyzer lib example && \
+    tar czvf sockjs_client.pub.tgz LICENSE README.md pubspec.yaml analysis_options.yaml lib/ && \
```

Comment:
Is there a reason we package analysis_options?

Reply:
it's generally what we should be doing. This way we *could* run proper analysis on the published artifact.